src/Stripe.net/Entities/_base/StripeEntity.cs

```diff
@@ -5,6 +5,7 @@ namespace Stripe
     using System.Runtime.CompilerServices;
     using Newtonsoft.Json;
 
+    [JsonObject(MemberSerialization.OptIn)]
```

Comment:
usual "I don't know what this means, are you sure we won't break anyone?" comment

Reply:
Right, sorry. [`MemberSerialization.OptIn`](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_MemberSerialization.htm) changes the behavior of Newtonsoft.Json like this:
> Only members marked with `JsonPropertyAttribute` [...] are serialized.

In other words, this causes members that _don't_ have an explicit Json attribute to be treated as if they were annotated with `JsonIgnore`. Since this PR also forces us to _not_ have any members without an explicit Json attribute, this is not really needed anyway, it's just an additional safety measure.